### ISSUE TITLE
Low Confirmation Rate

### ISSUE THREAD

**Issue by vuapo-eth** [mikrohash/isf-jclient#15]
The confirmation rate of the spammer (currently 15%) is significantly lower than the average confirmation rate of the tangle (25%). Considering that right now ~80% of spam is created by iotaspam.com spammers, the confirmation rate of transactions that are not created by iotaspam.com spammers sits at approximately 65%:

        ISF         NOT ISF      TOTAL
    (80% * 15%) + (20% * 65%) =   25%

Finding the cause for that and improving the isf-jclient confirmation rate, would vastly improve the confirmation rate of the tangle as a whole, since right now the majority of transaction is created by this spammer.

![conf php](https://user-images.githubusercontent.com/33261108/36866877-d48ce8ca-1d93-11e8-8326-b56472aa8e86.png)

**Comment by PJanisio** [comment]
Well if I correctly understood, please correct me if im wrong - actual spamming is not effective enough and you are now searchiing for this difference of 10% between tangle and iotaspam confirmation rate?

**Comment by vuapo-eth** [comment]
Yes, there seems to be something wrong with either the code of the spammer itself or the environment in which our users run it (which nodes they connect to etc.). Other spammers are way more effective in terms of confirmation rate.

**Comment by vuapo-eth** [comment]
for the first time, it's the other way around, weird :thinking: 

![conf php](https://user-images.githubusercontent.com/33261108/39446028-5b8419b8-4cbd-11e8-8fc9-dfe3bf4825d2.png)

**Comment by PJanisio** [comment]
Is it from logical view even possible?

**Comment by vuapo-eth** [comment]
Depends on what's the actual reason for difference:

* it might have been a bug in the spamming software that was fixed with v1.0.13
* or it might have been a single user who was using bad nodes and currently isn't spamming or changed to other nodes
* maybe it's just a random fluctuation and the blue line will be below the red line again soon

There are countless other possible explanations, we don't know what it is yet.
